script.readme

```diff
@@ -0,0 +1,6 @@
+**Code Desription**
+The script starts with running codes for cleaning a raw(bronze) customer and order data sets into an intermediate table(silver data) for further business analysis.
+Afterwhich, the silver table is then used to carry out the fofflowing analysis
```

Comment:
Very nice!

Reply:
More corrections made to the readme file. Thank you for the clarification.